packages/app/features/home/screen.tsx

```diff
@@ -1,32 +1,38 @@
 import { Text, useSx, View, H1, P, Row, A } from 'dripsy'
-import { useCallback, useState } from 'react'
+import { useCallback, useEffect, useState } from 'react'
 import { TextLink } from 'solito/link'
 import { MotiLink } from 'solito/moti'
 import SearchBar from './searchBar'
+import ResultDisplay, { parseResponse, ResultType } from './resultDisplay'
 import styles from './styles';
 
+const mockResults : ResultType[] = [
+  {word:'abacate', phonetic:'/abacate/'},
+  {word:'banana', phonetic:'/banana/'}
+];
+
 export function HomeScreen() {
-  const sx = useSx()
-  const [searchResults, setSearchResults] = useState([])
+  const [searchResults, setSearchResults] = useState<ResultType[]>(mockResults)
 
   const handleSearch = useCallback(async (searchText: String) => {
-    const response = await fetch(`https://api.dictionaryapi.dev/api/v2/entries/en/${searchText}`);
-    const data = await response.json();
-    console.log(data);
+    try {
+      const response = await fetch(`https://api.dictionaryapi.dev/api/v2/entries/en/${searchText}`);
+      const results = await parseResponse(response);
+      setSearchResults(results);
+    } catch(e) {
+      setSearchResults([]);
+    }
   },[]);
 
+
   return (
-    <View
-      sx={{ flex: 1, justifyContent: 'center', alignItems: 'center', p: 16 }}
-    >
+    <View style={styles.container}>
       <H1 sx={{ fontWeight: '800' }}>English Dictionary</H1>
       <View style={styles.container}>
         <SearchBar onButtonPress={handleSearch} />
+        {searchResults.map(result => <ResultDisplay word={result.word} phonetic={result.phonetic} />)}
```

Comment:
Alsooo, I almost forgot this one hehe

Do you see the error message in the Console Log?
![image](https://user-images.githubusercontent.com/10969700/229263069-40f11f10-5045-4df3-9d34-cac2055209cc.png)


Do you know what it's about?

React has a descriptive paradigm, that means we only explain how a certain state must be rendered and React does all the dirty work for us. The transition between states can be complicated though, so React is basically telling us that it wont be able to do a good job unless we give it a `key` for each mapped element. If we specify the `key` to react it will be able to distinguish the elements of our list and in case we delete or add some it can figure things out for us.

So we only need to pass a `key` prop to your component, make the `key` something unique, it could be the `word` for now but I guess it's not unique, in a following PR you can change the key to the first description or something that is truly unique

Could you understand why it's important? It won't kill your application, but providing a key when React asks can make it more performant and have better transitions specially on applications where such lists change a lot... in your application it will be indiferent cus the list basically doesn't change

Reply:
Didn't understand how to fix it.
I've created a prop key on the component but the error reamains

Reply:
Don't bother too much with this error. If it's hard to fix move on, as I said in your use case this hardly matters.

You shouldn't create a prop called `key` ... the `key` prop is already from React framework, it exists on every component. If you create your own prop key you'll override the one from React.

Reply:
Ok, done, I will make the key be the description in the feat/meaning display